tutorials/Sessions.md

```diff
@@ -0,0 +1,80 @@
+# Sessions
+Stytch user sessions are identified by a session token and a JWT (JSON Web Token) that are authenticated on, and returned from, our authentication endpoints. The Stytch iOS SDK automatically persists these tokens and appends them to network requests as required, to make interacting with Stytch's authentication flows as simple as possible, and provides both Automatic and Manual session management helpers.
+
+## SDK Sessions At-A-Glance
+### Session Data Persistence
+The Stytch iOS SDK persists the session token and JWT to device by saving them to the Keychain, after encrypting them using AES256-GCM. The actual session and user data is stored in-memory only.
+
+### Automatic Session Management
+When the Stytch iOS SDK is initialized, it decrypts the persisted session token, if any, and makes a Sessions Authenticate call to ensure the session is still active/valid. If it is, it will automatically rehydrate the session and user data in memory; if not, it clears all persisted session tokens. On every authentication response, the SDK updates the in-memory and device-persisted session data with the latest data returned from the endpoint. 
+
+In addition, once the SDK receives a valid session, it begins an automatic "heartbeat" job that checks for continued session validity in the background, roughly every three minutes. This heartbeat does not extend an existing session, it merely checks it's validity and updates the local data as appropriate. 
+
+It is then good practice when using the Stytch iOS SDK to access any data that may have been updated via the heartbeat call through their cached values. Otherwise if you hold onto a reference of a response that contains authentication information in it, like token or user, you risk the those values returned in the response becoming stale. Values updated via the heartbeat are: `User`, `sessionToken`,  `sessionJwt` and `session`. Examples for retrieving these cached values are below. 
+
+### Manual Session Management and Observation
+The [Sessions client](../Sources/StytchCore/StytchClient/StytchClient+Sessions.swift) provides properties to retrieve the current session tokens; methods for authenticating, updating, and revoking sessions; and a flow/callback to listen for changes in session state.
```

Comment:
Are they called also `flows` in iOS? Shouldn't it be a `publisher` or something?